.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build on various envs
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize # the head branch is updated from the base branch, new commits are pushed to the head branch, or the base branch is changed
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+      - 'release/**'
+
+jobs:
+  build:
+    name: build
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [ '14', '16' ]
+        os: [ 'ubuntu-latest', 'windows-latest', 'macos-latest' ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@722adc63f1aa60a57ec37892e133b1d319cae598 # https://github.com/actions/checkout/releases/tag/v2.0.0
+
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@56899e050abffc08c2b3b61f3ec6a79a9dc3223d # https://github.com/actions/setup-node/releases/tag/v1.4.4
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: |
+          npm ci
```

Comment:
What do you think about trying both `npm ci` and `npm install`? They are very likely to result in different versions.

Reply:
Do you mean that we should test both ways of installing dependencies on every PR? E.g. add it to matrix in this workflow?

Reply:
Yes, I think so. Using `npm install` will give us an early warning if a newer version of a dependency that matches our semver ranges in `package.json` has a problem.

Reply:
Just one question - how much does GH charge us for running these workflows? Should I be concerned about it?

Reply:
> Just one question - how much does GH charge us for running these workflows? Should I be concerned about it?

According to GitHub docs:

> GitHub Actions usage is free for both public repositories and self-hosted runners.

So it is free, as this is a public repo. I am sure there is some unspoken limit,  but probably we are far from it.